assignments/closure.js

```diff
@@ -1,15 +1,37 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+const challenge1 = (first_name, last_name) => {
```

Comment:
You did a great job making some progress on this Closure material. I think you'll get it down with practice :+1: